### DISCUSSION TITLE
Why can't I use github.dev on my phone?

### DISCUSSION THREAD

**Discussion by OwO996** [community/community#74779] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I open github.dev on my phone, I'm always stuck at "Setting up your web editor". I use chrome software to open github.dev on my iPhone.

**Comment by prawi** [comment]
Clear Cache and Cookies: Sometimes, cached data can cause issues. Try clearing your browser's cache and cookies before accessing GitHub.dev again.
